docs/references/ic-interface-spec.md

```diff
@@ -596,7 +596,7 @@ The Internet Computer has two HTTPS APIs for canister calling:
 
 3.  The IC asks the targeted canister if it is willing to accept this message and be charged for the expense of processing it. This uses the [Ingress message inspection](#system-api-inspect-message) API for normal calls. For calls to the management canister, the rules in [The IC management canister](#ic-management-canister) apply.
 
-4.  At some point, the IC may accept the call for processing and set its status to `received`. This indicates that the IC as a whole has received the call and plans on processing it (although it may still not get processed if the IC is under high load). Furthermore, the user should also be able to ask any endpoint about the status of the pending call.
+4.  At some point, the IC may accept the call for processing and set its status to `received`. This indicates that the IC as a whole has received the call and plans on processing it (although it may still not get processed if the IC is under high load). At the time of this transition, the call has not yet expired. Furthermore, the user should also be able to ask any endpoint about the status of the pending call.
```

Comment:
I find point 8 a bit confusing as well:

> In the case that the call has been retained for long enough, but the request has not expired yet

Can we be specific on "long enough" and "has not yet expired"?

Stupid question: does this relate to the "ingress_expiriry" field?

Reply:
Yes, this relates to the ingress_expiry field. 
I added  a few sentences about subnet time and call expiry above. Does this make it more clear what "has not expired" means here? If not, can you make a suggestion?

(let's discuss "long enough" in L646/647)

Reply:
Not introduced by this PR, but what does "the user should be able to ask any endpoint about the status of the pending call" mean here? Doesn't the user have to go to `read_state` endpoint?